src/Week1/HW01.hs

```diff
@@ -37,15 +58,20 @@ doubleEveryOther lst = helper 0 lst []
 
 -- Calculate the sum of all the digits in every Integer.
 sumDigits :: [Integer] -> Integer
-sumDigits lst = sum lst 
+-- Performs in one pass
+sumDigits = foldr (helper) 0
+    where
+        helper :: Integer -> (Integer -> Integer)
```

Comment:
Nitpick: function arrow is right associative, so you can leave  out these parens

```suggestion
        helper :: Integer -> Integer -> Integer
```